src/Microsoft.Azure.WebJobs.Protocols/ParameterLog.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Azure.WebJobs.Host.Protocols
 #if PUBLICPROTOCOL
     public class ParameterLog
 #else
-    internal class ParameterLog
+    public class ParameterLog // $$$ ok? Access from Extensions
```

Comment:
The host cross compiles these types, so shouldn't need visibility changes? I think you should just cross compile this into the new storage extension, following the WebJobs.Host/Protocols pattern